modules/runners/templates/start-runner.sh

```diff
@@ -60,6 +62,16 @@ chown -R $run_as .
 echo "Configure GH Runner as user $run_as"
 sudo --preserve-env=RUNNER_ALLOW_RUNASROOT -u "$run_as" -- ./config.sh --unattended --name "$instance_id" --work "_work" $${config}
 
+tee /opt/actions-runner/.setup_info <<EOL
+[
+  {
+    "group": "Machine AMI Id",
+    "detail": "$ami_id"
+  }
+]
+EOL
```

Comment:
Adding a bit more info, and similar to GH hosted runners
```suggestion
info_arch=$(uname -p)
info_os=$(( lsb_release -ds || cat /etc/*release || uname -om ) 2>/dev/null | head -n1 | cut -d "=" -f2- | tr -d '"')

tee /opt/actions-runner/.setup_info <<EOL
[
  {
    "group": "Operating System",
    "detail": "Distribution: $info_os\nArchitecture: $info_arch"
  },
  {
    "group": "Runner Image",
    "detail": "AMI id: $ami_id"
  }
]
EOL
```

Reply:
Example:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/11609620/194533649-44c43089-376b-41be-943d-7fc3dd3a4d45.png">